### PR TITLE
Create QuranFlash.com.xml

### PR DESCRIPTION
**Pull request by 6u621jpq6ybdw**

### CODE REVIEW

**Comment by J0WI** [comment]
This site has mixed content issues. Please also check for additional subdomains on this host.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

File: src/chrome/content/rules/QuranFlash.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Invalid Certificate:
+		v2.quranflash.com
```
Comment (J0WI) [inline src/chrome/content/rules/QuranFlash.com.xml]: See also `v1` and `v3`.
Please add a hint about the wildcard DNS entry on this host.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

**Pull request merged**